components/header.html

```diff
@@ -10,7 +10,7 @@
 <body>
     <header id="home">
         <nav class="navbar">
-        <div class="container">
+        <div class="containe">
```

Comment:
do you mean `container`?

Reply:
The container zainab used was conflicting with mine, so i removed the r pending the time zainab will work on the new header

Reply:
I used a bootstrap container, and zainab also had a container styled for the header.
Though she said she had resolved it. just waiting for it to be merged and i change my header